README.md

```diff
@@ -118,6 +118,25 @@ we copied.)
     // object containing the results by name.
     promise.then(function(response) {
       console.log(response.namedResult);
+    })
+    // If the remote object throws an rpc exception, the promise will be
+    // rejected. The error returned will have a property `kjType`, which
+    // will be a string representation of the exception's `type` field.
+    .catch(function(error) {
+        switch(error.kjType) {
+        case 'failed':
+            console.log("A generic problem occurred");
```

Comment:
Hmm, maybe should log the message so that people don't get the misimpression that there is no message?

    console.log("A generic problem occurred:", error.message);